inttest/common/footloosesuite.go

```diff
@@ -1444,3 +1445,37 @@ func (s *FootlooseSuite) AssertSomeKubeSystemPods(client *kubernetes.Clientset)
 
 	return false
 }
+
+type IPV6Config struct {
+	IPv6        bool   `json:"ipv6"`
+	FixedCidrV6 string `json:"fixed-cidr-v6"`
+}
+
+func (s *FootlooseSuite) ConfigureIPv6Docker(enabled bool) error {
+	config := IPV6Config{
+		IPv6:        enabled,
+		FixedCidrV6: `2001:db8:1::/64`,
+	}
+	json, err := json.Marshal(config)
+	if err != nil {
+		return fmt.Errorf("failed to serialize docker config file: %w", err)
+	}
+	tmpFile, err := ioutil.TempFile("/tmp", "daemon.json")
+	defer os.Remove(tmpFile.Name())
+	if err != nil {
+		return fmt.Errorf("failed to write temporary file: %w", err)
+	}
+	err = ioutil.WriteFile(tmpFile.Name(), json, 0644)
+	if err != nil {
+		return fmt.Errorf("failed to write to temporary file: %w", err)
+	}
+	err = exec.Command("sudo", "mv", "-f", tmpFile.Name(), "/etc/docker/daemon.json").Run()
+	if err != nil {
+		return fmt.Errorf("failed to write daemon.json file: %w", err)
+	}
+	err = exec.Command("sudo", "systemctl", "reload", "docker").Run()
```

Comment:
```suggestion
	err = exec.Command("sudo", "systemctl", "restart", "docker").Run()
```

IIRC `reload` just reloads the unit from disk but does NOT actually restart it

Reply:
sure we can try that
i folloed https://docs.docker.com/config/daemon/ipv6/